### PR TITLE
[consumption] add process niceness extension

### LINKED ISSUE #449: [consumption] process niceness extension
With PHP it is doable with just one line:
```
proc_nice($priority);
```
Where priority is integer from -20 to 19, but only values from 0 onwards is allowed without root permissions. Default is 0.

http://php.net/manual/en/function.proc-nice.php

Would be very usefull for environments with limited CPU power and where (in terms of CPU time) some consumers must overcome other not so important processes.

### PR DESCRIPTION
**Pull request by ramunasd**
fix #449

### CODE REVIEW

File: pkg/enqueue/Consumption/Extension/NicenessExtension.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public function __construct($niceness)
+    {
+        if (false === is_int($niceness)) {
+            throw new \InvalidArgumentException(sprintf(
+                'Expected niceness value is int but got: "%s"',
+                is_object($niceness) ? get_class($niceness) : gettype($niceness)
+            ));
+        }
+
+        if (0 !== $niceness) {
```
Comment (makasim) [inline pkg/enqueue/Consumption/Extension/NicenessExtension.php]: this has to be done `onStart`

File: pkg/enqueue/Symfony/Consumption/LimitsExtensionsCommandTrait.php [hunk 2]
```diff
@@ -58,6 +60,11 @@ protected function getLimitsExtensions(InputInterface $input, OutputInterface $o
             $extensions[] = new LimitConsumerMemoryExtension($memoryLimit);
         }
 
+        $niceness = (int) $input->getOption('niceness');
```
Comment (makasim) [inline pkg/enqueue/Symfony/Consumption/LimitsExtensionsCommandTrait.php]: we should not force int but check if the option is number and if not throw an exception with the message that explains it shuld be int.

Comment (ramunasd) [reply pkg/enqueue/Symfony/Consumption/LimitsExtensionsCommandTrait.php]: The check already is in constructor of extension. Do we really need additional one?

File: pkg/enqueue/Consumption/Extension/NicenessExtension.php (hunk 1 above)
Comment (ramunasd) [reply pkg/enqueue/Consumption/Extension/NicenessExtension.php]: fixed

**Pull request merged**